airbyte-integrations/bases/base-typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/TypeAndDedupeOperationValve.java

```diff
@@ -16,9 +16,6 @@
 public class TypeAndDedupeOperationValve extends ConcurrentHashMap<AirbyteStreamNameNamespacePair, Long> {
 
   private static final long NEGATIVE_MILLIS = -1;
-  private static final long FIFTEEN_MINUTES_MILLIS = 1000 * 60 * 15;
-  private static final long ONE_HOUR_MILLIS = 1000 * 60 * 60 * 1;
-  private static final long TWO_HOURS_MILLIS = 1000 * 60 * 60 * 2;
   private static final long FOUR_HOURS_MILLIS = 1000 * 60 * 60 * 4;
```

Comment:
nit: 6 hours? (though I don't think it really matters tbh)

Reply:
:doh-homer-emoji: